keyboards/shangming/dymium65/info.json

```diff
@@ -0,0 +1,138 @@
+{

```

Comment:
This is not an info.json.  This appears to be a via json file.   This needs to be removed and populated with the correct contents.